package.json

```diff
@@ -9,8 +9,8 @@
   "scripts": {
     "test": "standard 'src/**/*.js' 'test/**/*.js' storages.js constants.js && BABEL_ENV=commonjs ava",
     "test:watch": "npm test -- --watch",
-    "copy-types": "cp -r type-definitions/ lib/",
-    "clean": "rimraf lib dist es",
+    "copy-types": "rsync -av --exclude=constants.d.ts --exclude=storages.d.ts type-definitions/ lib/ && cp type-definitions/constants.d.ts type-definitions/storages.d.ts ./",
```

Comment:
Not really, I'm afraid, if you want to be fully compatible with typescript 2.x definitions. It is possible to use it now as single bundle file of the structure that @morlay created but it is now discouraged and likely will be deprecated in some future versions of typescript.